src/js/hooks/cycleCount/useCountStepTable.jsx

```diff
@@ -269,7 +269,8 @@ const useCountStepTable = ({
 
   const columns = [
     columnHelper.accessor(
-      (row) => (row?.binLocation?.label ? row?.binLocation : row.binLocation?.name), {
+      // TODO: This doesn't seem to do anything?????? How to change the value???
+      (row) => getBinLocationToDisplay(row?.binLocation), {
```

Comment:
:thinking:

Reply:
Those cells that you can see in the table come from the "cell" component created in our "defaultColumn" definition (that's the code that I wanted to split into a separate component, currently it's :spaghetti:). Try to use your function  within the code block starts with 
![image](https://github.com/user-attachments/assets/5dc89973-b5c5-4cca-bb6d-4732b9e69756)

Reply:
I poked around with it a bit more and still wasn't able to figure it out. I can't simply set the value of the TableCell Component to be getBinLocationToDisplay(value) because we only want to do that if a value is selected(?) It works fine currently when selecting new values, but existing, non-custom rows don't get the prefix. I wonder if it needs to be set differently in the getFieldProps for the SelectField, but I can't figure out how.